zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -5793,9 +5793,8 @@ components:
           description: The field to sort by.
           type: string
         order:
-          description: The order in which to sort the field.
-          type: string
-          default: asc
+          description: The sort order.
+          $ref: "#/components/schemas/SortOrderEnum"
```

Comment:
🙃 I believe that siblings next to `$ref` are ignored in OpenAPI so you can remove the description here. @pepopowitz, that is the way it works, right? 😆 I am horribly confused by now 🙈

Reply:
This is true, when I use allOf I can remove the description ;) I was fixing this right now

Reply:
Alternatively, you can keep the description and move the `$ref` into an `allOf`.